index.html

```diff
@@ -129,10 +129,76 @@ <h3 class="mb-4 section-subtitle text-bold text-proxima">What are my accomplishm
       </div>
       
     </section>
+
+    <section id="testimonials" class="pt-5 pb-5">
+      <div class="container pt-4 pb-4">
+        <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials</h2>
+
+        <div id="carouselExampleIndicators" class="carousel slide mb-5" data-ride="carousel">
+          <ol class="carousel-indicators">
+            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active">
+              <img
+                class="d-block img-fluid rounded-circle"
+                src="images/person-1.jpg"
+                srcset="images/person-1@2x.jpg 2x"
+                alt="Person 1"
+              />
+            </li>
+            <li data-target="#carouselExampleIndicators" data-slide-to="1">
+              <img
+                class="d-block img-fluid rounded-circle"
+                src="images/person-2.jpg"
+                srcset="images/person-2@2x.jpg 2x"
+                alt="Person 2"
+              />
+            </li>
+          </ol>
+
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <div class="mx-auto" style="max-width: 70%;">
+                <p class="testimonial-text">Lorem ipsum dolor sit amet, at ipsum lucilius his, ex pro munere percipitur, at eos alia munere deleniti. Nec in everti omittantur. Ei sint munere vix. Ut est possit referrentur, cum eu fabellas philosophia. Saperet habemus noluisse eam cu, ea autem option sadipscing vel. Est ne oblique eripuit pertinacia, semper noluisse ei sea.</p>
+                <div class="d-flex justify-content-end">
+                  <div>
+                    <p class="person-name text-proxima text-20 mb-1">Oleks Tsukur</p>
+                    <p class="person-title text-proxima mb-0">Full-Stack Developer</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="carousel-item">
+              <div class="mx-auto" style="max-width: 70%;">
+                <p class="testimonial-text">Lorem ipsum dolor sit amet, at ipsum lucilius his, ex pro munere percipitur, at eos alia munere deleniti. Nec in everti omittantur. Ei sint munere vix. Ut est possit referrentur, cum eu fabellas philosophia. Saperet habemus noluisse eam cu, ea autem option sadipscing vel. Est ne oblique eripuit pertinacia, semper noluisse ei sea.</p>
+                <div class="d-flex justify-content-end">
+                  <div>
+                    <p class="person-name text-proxima text-20 mb-1">Vlada Boiko</p>
+                    <p class="person-title text-proxima mb-0">UI/UX Designer</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```

Comment:
Change `href="#carouselExampleIndicators"` so that it corresponds to the specific id